packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/tasks/ImageResultTask.kt

```diff
@@ -31,7 +31,7 @@ open class ImageResultTask(private val promise: Promise,
       val imageExporterHandler = object : Listener {
         override fun onResult(out: ByteArrayOutputStream?, width: Int, height: Int) {
           val response = Bundle().apply {
-            putString("uri", outputFile.toURI().toString())
+            putString("uri", outputFile.toURL().toString())
```

Comment:
```suggestion
            putString("uri", outputFile.toURI().toURL().toString())
```
See the other comment 😉